starboard/android/shared/aosp_main.cc

```diff
@@ -0,0 +1,366 @@
+// Copyright 2018 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <dirent.h>
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <atomic>
+
+// #include "game-activity/GameActivity.h"
```

Comment:
Well. With these changes all of `aosp` stuff is basically unbuilt, untested code. Do we need it, now ? Or can we add it back in when we are actually building it ( 25lts branch isn't going anywhere, we can always take it from there )

Assumptions like "IS_EVERGREEN_COMPATIBLE" and such in this file don't even make sense with AOSP context.

Reply:
Could we just leave a giant NOTIMPLEMENTED() in the AOSP files?

Reply:
Merged aosp_main.cc and android_main.cc with #SB_IS(EVERGREEN_COMPATIBLE) macro

Reply:
Removed aosp files from this PR and into its own one: https://github.com/youtube/cobalt/pull/4436/files

We can unblock this one and continue the discussion there.